README.md

```diff
@@ -1 +1,5 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse!
+
+> This project is to understand about linters and gitworkflow with simple hello-microverse html text. 
```

Comment:
- [ ] I suggest you make use of this [template](https://github.com/microverseinc/readme-template) and add descriptive README information about your project. This is so that visitors of your repository know right away what the project is all about.